src/components/BooksList.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Book from './Book';
+
+const BooksList = state => {
+  const books = state.books.map(book => (
+    <Book
+      key={book.id}
+      id={book.id}
+      title={book.title}
+      category={book.category}
+    />
```

Comment:
pass the whole object

Reply:
Relocate this component into the `src/containers/` folder,  [ see requirements ]